packages/task/src/browser/quick-open-task.ts

```diff
@@ -40,25 +41,22 @@ export class QuickOpenTask implements QuickOpenModel, QuickOpenHandler {
 
     /** Initialize this quick open model with the tasks. */
     async init(): Promise<void> {
-        this.items = [];
-
-        const configuredTasks = await this.taskConfigurations.getTasks();
-        for (const task of configuredTasks) {
-            this.items.push(new TaskRunQuickOpenItem(task, this.taskService, false));
-        }
-
+        const configuredTasks = this.taskConfigurations.getTasks();
         const providedTasks = await this.taskService.getProvidedTasks();
-        for (const task of providedTasks) {
-            this.items.push(new TaskRunQuickOpenItem(task, this.taskService, true));
-        }
+
+        this.items = [];
+        this.items.push(
+            ...configuredTasks.map((t, ind) => new TaskRunQuickOpenItem(t, this.taskService, true, ind === 0 ? 'configured' : undefined)),
+            ...providedTasks.map((t, ind) => new TaskRunQuickOpenItem(t, this.taskService, false, ind === 0 ? 'provided' : undefined))
+        );
     }
 
     async open(): Promise<void> {
         await this.init();
         this.quickOpenService.open(this, {
             placeholder: 'Type the name of a task you want to execute',
             fuzzyMatchLabel: true,
-            fuzzySort: true
+            fuzzySort: false
```

Comment:
Hello @elaihau 
While checking it I've spotted different behavior of the tasks quick open depending on whether it's opened with the `Run Task...` command or with the `task ` quick open handler. Here's how it looks for me when I open it via `task ` quick open handler:
![screenshot from 2018-12-21 17-00-45](https://user-images.githubusercontent.com/1636395/50348982-0537c700-0543-11e9-8b24-660712576462.png)

To make it the same, it's better to disable fuzzy sort in [`getOptions`](https://github.com/theia-ide/theia/blob/master/packages/task/src/browser/quick-open-task.ts#L72) method and pass that options into `open` method, i.e. `this.quickOpenService.open(this, this.getOptions());`
WDYT?